client/service/src/config.rs

```diff
@@ -58,8 +61,12 @@ pub struct Configuration {
 	pub state_cache_size: usize,
 	/// Size in percent of cache size dedicated to child tries
 	pub state_cache_child_ratio: Option<usize>,
-	/// Pruning settings.
-	pub pruning: PruningMode,
+	/// State pruning settings.
+	pub state_pruning: PruningMode,
+	/// Block pruning settings.
```

Comment:
I'm fine with the naming, but we should improve the docs
```suggestion
	/// Number of blocks to keep in the db.
```